### PR TITLE
add mutation data to TermdbTest

### PR DESCRIPTION
**Pull request by congyu-lu**
## Description
add mutation data to TermdbTest. Only include one file (sampleName 3416 / sampleID 96 ) for methylationArrayNormalSubtracted and methylationArrayNormalSubtracted_bins as these files are very big (300-500kb for a methylationArrayNormalSubtracted_bins file and a few MB for a methylationArrayNormalSubtracted file). 

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: server/dataset/termdb.test.ts [hunk 1]
```diff
@@ -28,86 +28,218 @@ reason:
...
-export default <Mds3>{
-	isMds3: true,
-	cohort: {
-		db: {
-			file: 'files/hg38/TermdbTest/db'
-		},
-		allowedChartTypes: ['summary', 'survival', 'matrix', 'sampleScatter', 'cuminc', 'dataDownload'],
-		termdb: {
-			displaySampleIds: true, // allow to display sample-level data
+export default function () {
```
Comment (xzhou82) [inline server/dataset/termdb.test.ts:31]: why does it need to export a function?

File: server/test/tp/files/hg38/TermdbTest/mutationpersample/table [hunk 2]
```diff
@@ -0,0 +1,12 @@
+91	sdhanda/mb_portal/BT_database/TermdbTest/mutationpersample/91
```
Comment (xzhou82) [inline server/test/tp/files/hg38/TermdbTest/mutationpersample/table:1]: i don't think this `table` file is needed

File: server/dataset/termdb.test.ts (hunk 1 above)
Comment (congyu-lu) [reply server/dataset/termdb.test.ts:31]: It's used for both genomes (hg38 and hg38-test) in serverconfig.json

File: server/test/tp/files/hg38/TermdbTest/mutationpersample/table (hunk 2 above)
Comment (congyu-lu) [reply server/test/tp/files/hg38/TermdbTest/mutationpersample/table]: ok thanks removed.

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**